packages/devtools-launchpad/src/index.js

```diff
@@ -144,6 +143,12 @@ async function getTabs(actions) {
   actions.newTabs(firefoxTabs);
 }
 
+async function getTabs(actions) {
+  actions.clearTabs();
```

Comment:
My point is that getTabs is not trying to connect to new firefox clients. So, as I said, this forces you to refresh your browser after clicking on "Launch firefox".